server/src/test/java/server/haengdong/presentation/BillActionControllerTest.java

```diff
@@ -66,4 +71,26 @@ void saveAllBillAction1() throws Exception {
                 .andDo(print())
                 .andExpect(status().isBadRequest());
     }
+
+    @DisplayName("지출 내역을 삭제한다.")
+    @Test
+    void deleteBillAction() throws Exception {
+        String token = "TOKE11";
```

Comment:
```suggestion
        String token = "토다리 토큰";
```
c: 아래 부분도 수정해보면 어떨까요?